papers/ice_nucleation_2024/calibration.jl

```diff
@@ -100,7 +102,8 @@ function run_model(p, coefficients, IN_mode, FT, IC)
 
     # solve ODE
     local sol = run_parcel(IC, FT(0), t_max, params)
-    return sol[9, :] ./ (IC[7] + IC[8] + IC[9])  # frozen fraction
+    # return sol[9, end-25:end] ./ (IC[7] + IC[8] + IC[9])  # frozen fraction
+    return sol[9,end-25:end] ./ (IC[7] + IC[8] + IC[9])
```

Comment:
Might be good not to hard code 25 as the number but instead have some top level parameter that defines the window we average over at the end of the simulation. And then based on that compute the index value of the cutoff

Reply:
Will open as an issue if the other AIDA data are shorter than 100 timesteps long. Otherwise, I will have it as a variable called `end_sim` for now if that's fine